### ISSUE TITLE
Lord Rhyolith heroic

### ISSUE THREAD

**Issue by Sketish** [gamefreedomgit/Maelstrom#4343]
1.Concussive Stomp sometimes is spawning only 1 volcano instead of 2-3
![image](https://github.com/gamefreedomgit/Maelstrom/assets/122209088/fc0da435-b9c1-4b90-92d5-d6efd0bb6a85)
Also when comparing to retail videos, volcanos shouldnt spawn at edge between boss platform and lava surrounding him

2.Liquid Obsidium sometimes walk through textures and they should move a little bit slower
![image](https://github.com/gamefreedomgit/Maelstrom/assets/122209088/65841d40-7cdb-46cf-b2e5-ab16b03f1af3)

3.When boss is turning at the edge of platform it's hard to predict it's movement because he does 180 degree turn in 1-2 seconds.

4.Steering bar is not always matching boss turning direction, as example it's showing that boss will turn left and boss is turning right

5.DBM is not resetting after wipe

6.Sometimes boss is not reseting after whole raid is dead, and he just walks for 1-2 mins before random reset

7.Boss is verly slowly reacting when players are swapping from one leg to another( may be due to dampending on heroic)

8.Superheated is not being casted by boss after 5 min of encounter

9.Legs hitbox is changing during encounter, sometimes melee can hit from 15y and sometimes when the yare inside leg thay have error out of range

10.Fragments of Rhyolith are doing way mroe dmg from melee.
https://youtu.be/PJGlTVPu4v0?t=39
On that video theyu are hittign bear for 4,5k-5k and if we assume he has 60% reduced dmg from armor, each of them should hit for  around 12-12,5k

11.Molten armor buff should give 5% on heroic instead of 2%
![image](https://github.com/gamefreedomgit/Maelstrom/assets/122209088/dd435cdf-0d85-43cf-8d02-6e3c36fa3dde)
12. Boss should lose a stack of molten armor each time he does stomp or summon adds
![image](https://github.com/gamefreedomgit/Maelstrom/assets/122209088/5bf46a09-7ec6-4434-a284-df2d9a8cc1b4)

**Comment by voidwho** [comment]
Probably makes sense to take a look at https://github.com/gamefreedomgit/Maelstrom/issues/4340 I opened it. If anyone can confirm it or not(if it's an issue, or if something else is causing the steering issue). 
If yes, then probably it makes sense to link these 2 issues and work on them at the same time. 

At the moment - Steering is IMHO more critical then these other issues, as steering issue makes the fight unkillable until you get insane RNG.

**Comment by whothatdog** [comment]
> Probably makes sense to take a look at #4340 I opened it. If anyone can confirm it or not(if it's an issue, or if something else is causing the steering issue). If yes, then probably it makes sense to link these 2 issues and work on them at the same time.
> 
> At the moment - Steering is IMHO more critical then these other issues, as steering issue makes the fight unkillable until you get insane RNG.

what the question about the steering? If you possible to DoT both legs?
the answer is: yes.

we used to DoT both legs in many servers like Twin*

I am not sure **exactly** how the steering-mechanic works. 
what causing the legs to go <--- or ---> I **think** it is depend on the income damage (to the boss)
if group is doing 50k on right leg and 50.0001k on left leg the bar will start move to the left.

basically, you can dot both legs and what matters is how much damage the group pull.
if the group switch from one leg to an other leg once the damage reach to be higher than it the bar is changing.

**Comment by UltraNix** [comment]
Fixed.
